firmware/src/widgets/weatherwidget/WeatherWidget.h

```diff
@@ -68,5 +68,9 @@ class WeatherWidget : public Widget {
 
     const int MODE_HIGHS = 0;
     const int MODE_LOWS = 1;
+
+    int switchinterval = HIGH_LOW_INTERVAL;
```

Comment:
Maybe add
```
#ifndef HIGH_LOW_INTERVAL
  #define HIGH_LOW_INTERVAL 0
#endif
```
on top of the file to set a default value in case it's not defined in config.h?